Added changelog, prepared release (#45)
exasol/udf-mock-python: .github/pull_request_template.md

### All Submissions:

> * [ ] Is the title of the Pull Request correct?

Reviewer: ```suggestion
* [ ] Is the title of the pull request correct?
```

* [ ] Is the title of the corresponding issue correct?
* [ ] Have you updated the changelog?

> * [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../../pulls) for the same update/change? <!-- markdown-link-check-disable-line --> 

Reviewer: ```suggestion
* [ ] Have you checked to ensure there aren't other open [pull requests](../../../../pulls) for the same update/change? <!-- markdown-link-check-disable-line --> 
```

> * [ ] Are you mentioning the issue which this PullRequest fixes ("Fixes...")

Reviewer: ```suggestion
* [ ] Are you mentioning the issue which this pull request fixes ("Fixes...")
```
